examples/src/index.tsx

```diff
@@ -60,6 +60,7 @@ class App extends React.Component<{}, State> {
   render() {
     return (
       <div className="App">
+        <style>{styles}</style>
```

Comment:
Why do we have to change the example? I think we should aim to keep the previous version working.

If I understand correctly, this change is not needed.

Reply:
We actually dont have to change this, main reason was that i needed to change the webpack config to inline the css, and so this would fail because it also got inline, but i can create 2 different webpack rules for the src and example so we can keep the current behavoir and dont need to change the example ;)